File: Stones on the Table/stones_on_the_table.cpp
#include <iostream>
#define ll long long
using namespace std;

int Stones_on_the_Table()
{
    ll total_stone;
    cin>>total_stone;
    string color_of_stone;
    cin>>color_of_stone;
    ll stone_remain_in_row = 1;
    char prev = color_of_stone[0];
    for(int i = 1;i < total_stone;i++)
    {
        if(color_of_stone[i] != prev)
        {  stone_remain_in_row++;
           prev = color_of_stone[i] ;
        }
    }
    
    return (total_stone - stone_remain_in_row);

}
int main() {
	int remained_test_cases = 1 ;
// 	cin>>remained_test_cases;
	while(remained_test_cases > 0)
	{   cout<<Stones_on_the_Table();
	    remained_test_cases--;
	}
	return 0;
}

File: Stones on the Table/stones_on_the_table.py
def Stones_on_the_Table():
    
    total_stones = int(input())
    color_of_stone = input()
    stones_remain_in_row = 1 
    prev_stone = color_of_stone[0]
    
    for st in range(1,total_stones):
        if color_of_stone[st] != prev_stone:
            stones_remain_in_row += 1 
            prev_stone = color_of_stone[st]
            
    return total_stones - stones_remain_in_row        
    
            
    
    
remained_test_cases = 1 
# remained_test_cases = int(input())
while remained_test_cases > 0:
    print(Stones_on_the_Table())
    remained_test_cases -= 1 